### ISSUE TITLE
[High]: Chrome+Jaws: Screen reader: Role and label are not defined for controls.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8434]
## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Navigate to Any 'chat room'.
3. Navigate to the "Actions", "Emoji" and "Send" controls using the screen reader.
4. Observe the screen reader announcement.

## Expected Result:
When a role is defined appropriately for the interactive element, the screen reader user will interact with the element effectively.

## Actual Result:
Role is not defined for the "Actions", "Emoji" and "Send" controls. This made it difficult for screen reader users to access the functionality associated with it. Additionally, the label is not defined for the mentioned interactive element in the source code of the page.

**Other Occurrences:**
Similar issue repro on PRs #8195, #8117, #8340, #8443, #8521, #8585, #8672, #8634, #8828, #8839, #8844, #9025, #9026, #9132, #9147, #9022, #9284, #8299, #8928, #9180, #9276, #9315, #9343, #8569

## Workaround:
Yes

## Area issue was found in:
Chat room

## Failed WCAG checkpoints
4.1.2, 1.1.1

## User impact:
Screen reader users will not be made aware that these elements are buttons.

## Suggested resolution:
Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:
- Specify role="button" for the 'div' element containing the "Actions", "Emoji" and "Send" controls.
- Provide aria-label value such as "Actions", "Emoji" and "Send" for the 'div' element containing the "Actions", "Emoji" and "Send" controls.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.49-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Reference link:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
**Issue reported by:** Sumit

![#8091_Role and label not defined](https://user-images.githubusercontent.com/98817464/161214387-0750fa11-665b-4576-8cb2-81362fc16334.png)
